tests/translator/input/api_with_binary_media_types.yaml

```diff
@@ -1,8 +1,12 @@
+Parameters:
+  BMT:
+    Type: String
+    Default: 'image~1/jpg'
```

Comment:
Why `~1/`?

Reply:
I thought I'd fixed that. One sec, will push from local.